### DISCUSSION TITLE
node --watch with ts-node?

### DISCUSSION THREAD

**Discussion by aminroosta** [TypeStrong/ts-node#1955] [General]
Is it possible to use the experimental `--watch` flag with `ts-node`?
https://nodejs.org/api/cli.html#--watch

```bash
$ node --watch index.js
$ node --test --watch
```

**Comment by yangliguo7** [comment]
Yes.  I have the same question .  Do u konw how to solve the problem ?

I check the doc . maybe It dont support `watch`

**Comment by Khaaz** [comment]
Also very interested by this. Always looking to reduce the number of external tooling needed.
